src/data/actions/table.js

```diff
@@ -80,23 +79,35 @@ const paginateTable = (tableId, fetchMethod, pageNumber) => (
   }
 );
 
-const sortData = (data, ordering) => {
-  const direction = ordering && ordering.indexOf('-') !== -1 ? 'desc' : 'asc';
-  const column = (ordering && ordering.replace('-', ''));
-
-  // `parseKeyValue` adjusts the key's value into its appropriate data type to ensure
-  // proper sorting and sort order (e.g., asc/desc). A numeric value (even if passed in
-  // as a string) must be parsed as an actual numeric value. An empty value (e.g., null,
-  // undefined) must be parsed as an empty string to ensure the empty values are forced
-  // to the top in an ascending sort order.
-  const parseKeyValue = (obj) => {
-    const value = obj[column] || '';
-    if (!Number.isNaN(value) && !Number.isNaN(parseFloat(value))) {
-      return parseFloat(value);
+const sortBy = (dataToSort, orderField) => {
```

Comment:
This is probably fine for now but we may want to move it to utils as I can see it being used in other action handlers in the future.

Reply:
Agreed 👍